internal/thelma/utils/testutils/fake_token_source.go

```diff
@@ -0,0 +1,17 @@
+package testutils
+
+import "golang.org/x/oauth2"
+
+type fakeTokenSource struct {
+	fakeToken string
+}
+
+func NewFakeTokenSource(token string) oauth2.TokenSource {
```

Comment:
is this specifically a mock for an oauth2 access token?

Reply:
yep, basically a mock for the oauth2.TokenSource, except instead of generating a testify mock for it, we're creating a fake implementation